### PR TITLE
Mirror feature/utf8string branches

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -577,10 +577,13 @@
         "https://github.com/dotnet/cli/blob/release/**/*",
         "https://github.com/dotnet/corefx/blob/master/**/*",
         "https://github.com/dotnet/corefx/blob/release/**/*",
+        "https://github.com/dotnet/corefx/blob/feature/utf8string/*",
```
Comment (dagood) [inline Maestro/subscriptions.json]: `*` doens't work, `**/*` is actually specifically handled:
https://github.com/dotnet/versions/blob/efb1bc6623cbcac8c659bd0bf188e64535a38cf4/Maestro/src/Microsoft.DotNet.Maestro.WebApi/Models/SubscriptionsModel.cs#L89
https://github.com/dotnet/versions/issues/207 related for more flexibility.

Comment (mmitche) [reply Maestro/subscriptions.json]: Dangit...just remove then?

Comment (dagood) [reply Maestro/subscriptions.json]: Just replace the `*`s with `**/*` and this change LGTM.

**Comment by joshfree** [comment]
Thanks @mmitche @dagood 

/cc @GrabYourPitchforks

**Pull request merged**